### ISSUE TITLE
Disabilities pages accessibility audit

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94610]
## Issue Description
Conduct an accessibility audit of the following pages:

claim-type
disabilities/orientation
disabilities/rated-disabilities
new-disabilities/add
new-disabilities/follow-up
new-disabilities/ptsd*
toxic-exposure
pow
additional-disability-benefits
adaptive-benefits
aid-and-attendance
individual-unemployability
additional-disability-benefits-summary
disabilities/summary

---
## Tasks
- [ ] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [ ] Record bugs & violations in Google Sheets

## Acceptance Criteria
- [ ] An accessibility audit is performed for the pages listed above
- [ ] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review
